WordPress/Classes/Services/PostService+Revisions.swift

```diff
@@ -27,9 +27,7 @@ extension PostService {
                                         let revisions = self.syncPostRevisions(from: remoteRevisions ?? [],
                                                                                for: postId.intValue,
                                                                                with: blogId.intValue)
-                                        ContextManager.sharedInstance().save(self.managedObjectContext, completion: {
-                                            success(revisions)
-                                        }, on: .main)
+                                        ContextManager.sharedInstance().save(self.managedObjectContext, completion: success, on: .main)
```

Comment:
Would it be worth following up with removing the `ContextManager.sharedInstance()` access in favor of dependency injection?

Reply:
`PostService`, along with a few other types in the post-editing area, is quite complicated. It's not so easy to do the refactor of replacing the `NSManagedObjectContext` instance property with a `CoreDataStack` instance. So, we probably still need to wait for a while for that to happen...

Reply:
No worries